### PR TITLE
Use shared CoC and contribution guidelines, fix license in package.json

### PR DESCRIPTION
**Pull request by dominykas**
See https://github.com/nodejs/package-maintenance/issues/348

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "type": "commonjs",
   "author": "Wes Todd <wes@wesleytodd.com>",
-  "license": "ISC",
+  "license": "MIT",
```
Comment (dominykas) [inline package.json:8]: @wesleytodd I assume the `LICENSE` file is correct here?

**Comment by wesleytodd** [comment]
Some of these files are actually present in the `.github` directory.  I have been doing this because I get annoyed with 20 meta files at the root of my projects, and catching most of them with one `.npmignore` is nice.  I actually think we should look at using a `.github` repo for this.  That way we dont need to PR them all across all the repos.

That said, the `package.json` change for sure was part I missed when moving it here from my personal repo.

File: package.json (hunk 1 above)
Comment (wesleytodd) [reply package.json:8]: Yep, I changed the license when I moved it but forgot this!  Thanks!

**Comment by dominykas** [comment]
We might also be able to get away with a .github repo and leave them out in the individual repos, as has been suggested in the issue. I think it's probably best to discuss that over there.

**Comment by dominykas** [comment]
Updated this to rely on the CoC/contrib docs from the shared .github repo.

**Comment by dominykas** [comment]
Will open a new PR, based on latest guidelines.

**Pull request closed without merging**